content/intro-to-storybook/vue/zh-CN/simple-component.md

```diff
@@ -0,0 +1,281 @@
+  component: Task,
+  // Our exports that end in "Data" are not stories.
+  excludeStories: /.*Data$/,
+};
+
+export const actionsData = {
+  onPinTask: action('pin-task'),
+  onArchiveTask: action('archive-task'),
+};
+
+const Template = (args, { argTypes }) => ({
+  components: { Task },
+  props: Object.keys(argTypes),
+  methods: actionsData,
+  template: '<Task v-bind="$props" @pin-task="onPinTask" @archive-task="onArchiveTask" />',
+});
+
+export const Default = Template.bind({});
+Default.args = {
+  task: {
+    id: '1',
+    title: 'Test Task',
+    state: 'TASK_INBOX',
+    updatedAt: new Date(2018, 0, 1, 9, 0),
+  },
+};
+
+export const Pinned = Template.bind({});
+Pinned.args = {
+  task: {
+    ...Default.args.task,
+    state: 'TASK_PINNED',
+  },
+};
+
+export const Archived = Template.bind({});
+Archived.args = {
+  task: {
+    ...Default.args.task,
+    state: 'TASK_ARCHIVED',
+  },
+};
+```
+
+Storybook 有两个基本的组织级别：组件和他的 story。可以将每个 story 视作其组件的排列组合。您可以根据需要给每一个组件创建任意个 story。
+
+- **组件**
+  - Story
+  - Story
+  - Story
+
+我们创建了一个`default`来提示 Storybook 我们正在文档化的组件：
+
+- `component` -- 组件本身,
+- `title` -- 在 Storybook 应用侧边栏的显示,
+- `excludeStories` -- story 本身需要但是不用在 Storybook 应用中渲染的信息。
+
+我们为每一个我们需要测试的状态导出一个函数，以此来定义我们的 story。Story 实际上就是一个根据给定的状态返回已渲染元素的函数---就像是[无状态函数式组件](https://vuejs.org/v2/guide/render-function.html#Functional-Components)那样。
+
+因为我们的组件存在多种排列组合，所以设置一个`Template`变量不失为一种便捷的做法。使用这样的模式来创建您的 Story 可以大量减少代码量和维护成本。
+
+<div class="aside">
+💡 <code>Template.bind({})</code> 是 <a href="https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Function/bind">标准JavaScript</a> 中用来复制函数的技术。我们使用这项技术保证了在使用同一份实现的同时，让每一个导出的story可以配置自己的属性。
+</div>
+
+Arguments 或者简写[`args`](https://storybook.js.org/docs/vue/writing-stories/args) , 让我们可以在不重启 Storybook 的前提下实时编辑我们的组件。只要 [`args`](https://storybook.js.org/docs/vue/writing-stories/args) 的值被修改我们的组件也会相应的更新。
+
+我们创建了一个最基本的`task`变量来描述 task 组件应该呈现的样子。这通常是根据真实数据来进行建模的。同时，`导出`此 task 也让我们在以后编写其他 story 时可以复用它，详见下文。
+
+`action()`使我们可以创建一个回调函数，当点击事件触发时 Storybook UI 的**actions**面板会显示结果。所以如果我们创建了一个 pin 按钮，我们就可以通过面板清楚的知道按钮是否被成功点击了。
+
+考虑到我们需要为组件的每一个排列组合都传入同样的 actions，通常的便捷做法是将他们合并到一个`actionsData`变量中，并传入给每一个定义好的 story 中(story 使用`methods`属性访问)。
+
+值得一提的是当我们将组件所需的操作都合并到`actionsData`之后，我们可以在其他组件复用此组件时，让其他组件的 story 也可以复用`导出`的`actionsData`，详见下文。
+
+<div class="aside">
+💡 <a href="https://storybook.js.org/docs/vue/essentials/actions"><b>Actions</b></a>帮助您在隔离构建UI组件时验证交互。 通常情况下您不会持有应用程序上下文中函数和状态的访问权限。请使用<code>action()</code>保存它们。
+</div>
+
+## 配置
+
+我们需要对 Storybook 的配置做几处修改，这样其不仅可以识别到近期创建的 story，同时还允许我们可以使用应用的 CSS 文件（在`sec/index.css`）。
+
+如下修改您的 Storybook 配置文件(`.storybook/main.js`)：
+
+```diff:title=.storybook/main.js
+module.exports = {
++ stories: ['../src/components/**/*.stories.js'],
+  addons: ['@storybook/addon-links', '@storybook/addon-essentials'],
+};
+```
+
+完成上述的修改后，如下所示修改您`.storybook`文件夹中的`preview.js` ：
+
+```diff:title=.storybook/preview.js
+
+import '../src/index.css';
```

Comment:
@zydmayday if you don't mind adding the missing "+" here, in this case where adding the global styles to Storybook.